server/endpoint_urls/build_buddy_url/build_buddy_url.go

```diff
@@ -0,0 +1,31 @@
+package build_buddy_url
+
+import (
+	"flag"
+	"net/url"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/alert"
+	"github.com/buildbuddy-io/buildbuddy/server/util/flagutil"
+)
+
+var buildBuddyURL string
```

Comment:
Consider adding a function that will define the flag and return the string like the standard flag functions.

func URLVar() *string {
 ...
}